### PR TITLE
add ex14 folder

### PR DESCRIPTION
**Pull request by norona-marynicole**
I add my exe14 folder here

### CODE REVIEW

File: norona_marynicole/countries.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <body>
+    <div class="container">
+      <h1>Countries of the World</h1>
+      <input
+        type="text"
+        id="countryInput"
+        placeholder="Enter country name..."/>
+      <button onclick="searchCountry()">Search</button>
+      <div class="country-details" id="countryDetails"></div>
+      <div class="same-region-countries" id="sameRegionCountries"></div>
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/countries.html:18]: IDs should be written in snake case. All lower case separated by underscore

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: norona_marynicole/countries.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+function searchCountry() {
+  let countryName = document.getElementById("countryInput").value.trim();
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
+      "<p>Please enter a country name.</p>";
+    document.getElementById("sameRegionCountries").innerHTML = "";
+    document.getElementById("newCountryDetails").innerHTML = "";
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/countries.js:7]: IDs should be written in snake case. All lower case separated by underscore

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: norona_marynicole/countries.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+              country.name.nativeName
+                ? Object.values(country.name.nativeName).join(", ")
+                : "N/A"
+            }</p>
+            <p><strong>Official Languages:</strong> ${
+              country.languages
+                ? Object.values(country.languages).join(", ")
+                : "N/A"
+            }</p>
+          `;
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/countries.js:59]: When splitting the elements into multiple lines, make sure that the continuation line is indented.

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_11

File: norona_marynicole/countries.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      document.getElementById("countryDetails").innerHTML = details;
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/countries.js:60]: Ds should be written in snake case. All lower case separated by underscore

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: norona_marynicole/countries.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      let region = regionData[0].region;
+      let sameRegionCountriesList = regionData
+        .map(function (c) {
+          return `
+              <div class="country-card">
+                <img src="${c.flags.svg}" alt="Flag of ${c.name.common}"
+                 width="50">
+                <p>${c.name.common}</p>
+              </div>
+            `;
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/countries.js:80]: Remove unnecessary breaking of lines. Place line breaks in proper places

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27

File: norona_marynicole/countries.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+        })
+        .join("");
+      document.getElementById("sameRegionCountries").innerHTML = `
+            <h2>Countries in the Same Region (${region})</h2>
+            <div class="country-list">${sameRegionCountriesList}</div>
+          `;
+    })
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/countries.js:87]: 1. Remove unnecessary breaking of lines. Place line breaks in proper places. 
Follow the guideline
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27

2. IDs should be written in snake case. All lower case separated by underscore
Follow the guideline
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: norona_marynicole/countries.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    .catch(function (error) {
+      console.error("Error fetching data:", error);
+      document.getElementById("countryDetails").innerHTML =
+        "<p>An error occurred: " + error.message + "</p>";
+      document.getElementById("sameRegionCountries").innerHTML = "";
+      document.getElementById("newCountryDetails").innerHTML = "";
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/countries.js:93]: IDs should be written in snake case. All lower case separated by underscore

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: norona_marynicole/index.html [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+      <h2 class="opinion">MY OWN OPINION ON F2F OR ONLINE CLASS</h2>
+      <p>
+        In my perspective, conducting a face-to-face class last April 20, 2024,
+        was more convenient for understanding the lesson easily. It was
+        advantageous since not all students have necessary materials like
+        laptops, PCs, and internet access at home, especially for subjects like
+        coding. Face-to-face classes facilitate better understanding of
+        practical activities and provide necessary materials like PCs and
+        internet access. However, in deciding between face-to-face or online
+        classes, safety should be a priority, especially considering the current
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/index.html:111]: There should be max of 80 col per line.

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0

File: norona_marynicole/index.html [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+        <li>
+          <p>
+            Your goals for education, stability, financial security, and family
+            well-being reflect ambition and determination, promising a future of
+            success and fulfillment. - Bergado, Shamma
+          </p>
+        </li>
+        <li>
+          <p>
+            Your goals perfectly represent your values and desires. I'm thrilled
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/index.html:62]: There should be max of 80 col per line.

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0

File: norona_marynicole/index.html [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+        <li>To give a comfortable life to my family</li>
+        <li>To be rich and successful in life</li>
+      </ol>
+
+      <h2 class="title">COMMENTS OF MY TEAMMATES</h2>
+      <ul>
+        <li>
+          <p>
+            Your goals for education, stability, financial security, and family
+            well-being reflect ambition and determination, promising a future of
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/index.html:56]: There should be max of 80 col per line.

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0

File: norona_marynicole/script.js [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+  commentsSection.innerHTML = "<h2 class='comment'>COMMENTS</h2>";
+  comments.forEach((comment) => {
+    const commentDiv = document.createElement("div");
+    commentDiv.classList.add("comment");
+    const commentDate = new Date(comment.date).toLocaleString();
+    commentDiv.innerHTML = `
+                  <p>Name: ${comment.name}</p>
+                  <p>Comment: ${comment.text}</p>
+                  <p class="comment-date">Date: ${commentDate}</p>
+              `;
```
Comment (Reyes-PaulAngelo) [inline norona_marynicole/script.js:56]: You can have this on the same line.

`<p class="comment-date">Date: ${commentDate}</p>;`

**Review by Reyes-PaulAngelo — COMMENTED** [review]
Place all your assets into a common folder so it won't clutter your project directory.
     /image or /asset - contains all the images
     /js or /script - contains all the JS files
     /style - contains all the CSS files


Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g26e9a7bd30a_1_0

**Review by Reyes-PaulAngelo — APPROVED** [review]
you can now merge

**Comment by norona-marynicole** [comment]
Thank you!

**Pull request merged**